### ISSUE TITLE
Atom package documentation web page should handle relative image URLs.

### ISSUE THREAD

**Issue by thedaniel** [atom/atom.io#14]
_From @peterhal on July 9, 2015 18:1_

Github supports relative URLs for things like images in markdown files. The Atom generated package docs do not.

For example compare these:

https://atom.io/packages/nuclide-flow

And:

https://github.com/facebook/nuclide/tree/master/pkg/nuclide/flow

The relative image URLs work in the github page, but not the Atom page.

_Copied from original issue: atom/atom#7819_

**Comment by thedaniel** [comment]
_From @peterhal on July 9, 2015 18:2_

@thedaniel Thanks for taking a look

**Comment by thedaniel** [comment]
I filed an issue on the repo for the site, will close this when it's fixed.

**Comment by thedaniel** [comment]
_From @batjko on July 9, 2015 18:6_

Yea that one's been a trip-up for many a newbie package developer since the start.

**Comment by sindresorhus** [comment]
:+1:

**Comment by Alhadis** [comment]
Any progress?

I just stubbed my toe on this bug for the *second* time. [Once was enough](https://github.com/Alhadis/language-maxscript/commit/236fa8c0c80a0aba69f402ae0954392d99363b3f), but I still had to go and [do it again](https://github.com/Alhadis/language-apl/commit/076c57eb926515aa975eb14d70b4bb0e6e4811cc). ⍨

Being the scatter-brained idiot that I am, I'm sure I'll forget a second time when I get around to developing a language package for ROFF.

`#please-make-it-stop`

**Comment by Alhadis** [comment]
So, uh, I see this is still an issue...

<img width="759" alt="C'mon guys, seriously..." src="https://cloud.githubusercontent.com/assets/2346707/21526557/3f545aa6-cd79-11e6-8577-4c319abcea91.png">

This isn't exactly a trivial concern, particularly since it requires an entirely new release to be published just to fix a link if an author notices this too late.

**Comment by thedaniel** [comment]
It sounds like a trivial concern to me, if you can just push a patch release in 2 seconds and fix it - the world isn't running out of version numbers quite yet. (nb: i don't work on this anymore, this is a civilian opinion)

**Comment by Alhadis** [comment]
@thedaniel Some authors prefer to keep their package's release history clean and meaningful. Which means not having to cut a "patch" to work around a lazy oversight. Since this involves a permanent addition to a codebase's revision history, it isn't trivial if the author cares about clean version control.

This assumes a package author is even aware that an absolute URL needs to be used, instead of a relative one.

**Comment by jasonrudolph** [comment]
> Github supports relative URLs for things like images in markdown files. The Atom generated package docs do not.

We've updated atom.io to support image tags that use relative URLs to point to image files in the package's repository. With this update, atom.io should render your README's images in the same way that github.com renders your README's images.

Note that atom.io caches the rendered READMEs for performance. Because of that, existing package READMEs with relative image URLs will still exibit this issue. However, once a new version of the package is published, atom.io will properly render the README's images.

To demonstrate the fix, I manually expired the cache for the [language-file-magic](https://atom.io/packages/language-file-magic) package, and atom.io now correctly renders its images:

![screenshot](https://cloud.githubusercontent.com/assets/2988/25148843/492cddc8-244a-11e7-8409-05d7bd60294d.png)
